src/app/(dashboard)/perfil/page.tsx

```
@@ -1,14 +1,85 @@
+  const isLoading = stats === undefined;
+
+  const hasThemeData = stats && stats.byTheme && stats.byTheme.length > 0;
+
   return (
     <div className="container mx-auto p-6">
       <h1 className="mb-6 text-2xl font-semibold">Meu Perfil</h1>
 
+      {isLoading ? (
+        // Loading state
+        <div className="mb-8 grid grid-cols-1 gap-6 md:grid-cols-4">
+          {[...Array(4)].map((_, i) => (
```

ESLint

[error] Use `new Array()` instead of `Array()`.

(unicorn/new-for-builtins)